rules/cross-platform/execution_aws_ssm_sendcommand_with_command_parameters.toml

```diff
@@ -2,7 +2,9 @@
 creation_date = "2022/09/03"
 integration = ["endpoint", "auditd_manager"]
 maturity = "production"
-updated_date = "2024/09/23"
+updated_date = "2024/11/07"
+min_stack_comments = "New fields added: investigation_fields"
+min_stack_version = "8.11.0"
```

Comment:
We are already min stacking at 8.11.0 as supported version. So This might not be required as explicitly required. 
When we ran trim versions such instances were removed
```
0415258b-a7b2-48a6-891a-3367cd9d4d31: locked min_stack_version <= 8.11.0 - removing!, rule min_stack_version dropped
043d80a3-c49e-43ef-9c72-1088f0c7b278: 8.10 updated to: 8.11
053a0387-f3b5-4ba5-8245-8002cca2bd08: 8.10 updated to: 8.11
0564fb9d-90b9-4234-a411-82a546dc1343: 8.10 updated to: 8.11
05b358de-aa6d-4f6c-89e6-78f74018b43b: 8.10 updated to: 8.11
0635c542-1b96-4335-9b47-126582d2c19a: 8.10 updated to: 8.11
06568a02-af29-4f20-929c-f3af281e41aa: 8.10 updated to: 8.11
06a7a03c-c735-47a6-a313-51c354aef6c3: 8.10 updated to: 8.11
06dceabf-adca-48af-ac79-ffdf4c3b1e9a: 8.10 updated to: 8.11
074464f9-f30d-4029-8c03-0ed237fffec7: 8.10 updated to: 8.11
07b1ef73-1fde-4a49-a34a-5dd40011b076: 8.10 updated to: 8.11
0859355c-0f08-4b43-8ff5-7d2a4789fc08: 8.10 updated to: 8.11
0ab319ef-92b8-4c7f-989b-5de93c852e93: locked min_stack_version <= 8.11.0 - removing!, rule min_stack_version dropped
``` 

Check PR https://github.com/elastic/detection-rules/pull/4256

Reply:
Fixed, thanks Shashank!